cloud-sql/mysql/Startup.cs

```diff
@@ -219,6 +252,8 @@ DbConnection GetMySqlConnection()
                 app.UseGoogleExceptionLogging();
                 app.UseExceptionHandler("/Home/Error");
             }
+            // Create Database table if it does not exist.
+            InitializeDatabase();

```

Comment:
I would move this call to Program.cs after the host has been initialized. There's no guarantee that at this point in the code everything is up and running.

Reply:
Done.